packages/pieces/bettermode/src/index.ts

```diff
@@ -0,0 +1,24 @@
+
+import { createPiece } from "@activepieces/pieces-framework";
+import { bettermodeAuth } from "./lib/auth";
+import { createDiscussionAction } from "./lib/actions/create-discussion";
+import { createQuestionAction } from "./lib/actions/create-question";
+import { assignBadgeAction } from "./lib/actions/assign-badge";
+import { revokeBadgeAction } from "./lib/actions/revoke-badge";
+
+export const bettermode = createPiece({
+  displayName             : "Bettermode",
+  auth                    : bettermodeAuth,
+  minimumSupportedRelease : '0.9.0',
+  logoUrl                 : "https://sandbox.joeworkman.net/bettermode.png",
```

Comment:
Change logoUrl to https://cdn.activepieces.com/pieces/bettermode.png (we'll upload it before merging)